internal/engine/actions/remediate/pull_request/pull_request.go

```diff
@@ -200,13 +248,24 @@ func (r *Remediator) Do(
 	var remErr error
 	switch remAction {
 	case interfaces.ActionOptOn:
-		alreadyExists, err := prWithContentAlreadyExists(ctx, r.ghCli, repo, magicComment)
+		alreadyExists, slug, err := prWithContentAlreadyExists(ctx, r.ghCli, repo, magicComment)
 		if err != nil {
 			return nil, fmt.Errorf("cannot check if PR already exists: %w", err)
 		}
 		if alreadyExists {
 			zerolog.Ctx(ctx).Info().Msg("PR already exists, won't create a new one")
-			return nil, nil
+			b, err := json.Marshal(map[string]any{
+				"status": db.RemediationStatusTypesPending,
+				"status_data": map[string]any{
```

Comment:
Would you want to implement a custom `MarshalJSON` method for `prSlug`, rather than writing the marshalling code here?

Reply:
done :+1: